src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h

```diff
@@ -135,20 +136,18 @@ namespace OpenMS
     "X:-C5H8O4-H2S"
   };
 
-  static constexpr std::array<const char*, 13> modifications_RNA_UV_6SG_EXTENDED
+  static constexpr std::array<const char*, 11> modifications_RNA_UV_6SG_EXTENDED
   {   
     "X:",
     "X:-H2O",
     "X:-H3PO4",
     "X:-HPO3",
     "X:-H2S",
-    "X:+HPO3-H2S", // P1 artefact?
     "X:-HPO3-H2S", //pRBSID
     
 
     "X:-C5H8O4",
     "X:-C5H8O4-H2O",
-    "X:-C5H8O4-H3PO4",
```

Comment:
"X:-C5H8O4-H3PO4",  => C5H3N5O-1S1  Artefact

Reply:
Not sure if I get this one

Reply:
"X:-C5H8O4-H3PO4", => C10H14N5O7P1S1-C5H8O4-H3PO4 leads to C5H3N5O-1S1 (negative O).

BTW C5H8O4 containing adducts found very few PSMs ( just 2,3), so we can remove it all? or we can go for generalization of our software, so we will keep it?

Reply:
"X:-C5H8O4-H2O", "X:-C5H8O4-HPO3", and "X:-C5H8O4-H2S" are very Rarely found for 6SG, or because of the HF effect.

Reply:
Hard to say… for the extended settings I think we could keep them. If they don’t negatively influence results …